bindings/python/examples/saveCCBToFile/saveCCBToFile.py

```diff
@@ -0,0 +1,112 @@
+#
+# BSD 3-Clause License
+#
+# Copyright (c) 2019, Analog Devices, Inc.
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# 1. Redistributions of source code must retain the above copyright notice, this
+#    list of conditions and the following disclaimer.
+#
+# 2. Redistributions in binary form must reproduce the above copyright notice,
+#    this list of conditions and the following disclaimer in the documentation
+#    and/or other materials provided with the distribution.
+#
+# 3. Neither the name of the copyright holder nor the names of its
+#    contributors may be used to endorse or promote products derived from
+#    this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+# FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+# OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+import aditofpython as tof
+import sys
+import time
+import os
+
+ccb_dir = '\\ccb_directory\\'
+ccb_prefix = 'ccb_'
+
+if __name__ == "__main__":
+
+    if len(sys.argv) < 2  or sys.argv[1] == "--help" or sys.argv[1] == "-h" :
+        print("save_ccb.py usage:")
+        print("USB / Local connection: save_ccb.py <config>")
+        print("Network connection: save_ccb.py <ip> <config>")
+        exit(1)
+
+    system = tof.System()
+
+    cameras = []
+    ip = ""
+    if len(sys.argv) == 3 :
+        ip = sys.argv[1]
+        config = sys.argv[2]
+        print (f"Looking for camera on network @ {ip}. Will use {config}.")
+        ip = "ip:" + ip
+    elif len(sys.argv) == 2 :
+        config = sys.argv[1]
+        print (f"Looking for camera on UVC. Will use {config}.")
+    else :
+        print("Too many arguments provided!")
+        exit(1)
+
+    dir_path = os.path.dirname( os.path.abspath(__file__)) + ccb_dir
+    if os.path.exists(dir_path):
+        print(f"The directory {dir_path} already exists.")
+    else:
+        # Create the directory
+        os.makedirs(dir_path)
+        print(f"The directory {dir_path} was created.")
+
+    status = system.getCameraList(cameras, ip)
+    print("system.getCameraList()", status)
+
+    camera1 = cameras[0]
+
+    status = camera1.setControl("initialization_config", config)
```

Comment:
This one is deprecated as well, you need to parse the config file in the initialize function:
https://github.com/analogdevicesinc/ToF/blob/master/sdk/include/aditof/camera.h#L67